libkbfs/file_data.go

```diff
@@ -1744,7 +1744,7 @@ func (fd *fileData) deepCopy(ctx context.Context, codec kbfscodec.Codec,
 
 	// Handle the single-level case first.
 	if !topBlock.IsInd {
-		newTopBlock, err := topBlock.DeepCopy(codec)
+		newTopBlock := topBlock.DeepCopy()
```

Comment:
Can you please remove `codec` from the parameters too?  I think it's unused now.